lerobot/scripts/train.py

```diff
@@ -170,6 +171,7 @@ def log_train_info(logger: Logger, info, step, cfg, dataset, is_offline):
         f"lr:{lr:0.1e}",
         # in seconds
         f"updt_s:{update_s:.3f}",
+        f"dloading_s:{dataloading_s:.3f}",
```

Comment:
Can we please be consistent with `dloading_s` vs `dataloading_s` (preferring the second one as it is clearer).

Reply:
This update is done.

Reply:
Looking good! Thank you

Reply:
Agreed on this. Good shout @Cadene